### PR TITLE
Create default image and thumbnail crops

### PR DESCRIPTION
**Pull request by zachgarwood**
This moves the aspect ration and cropping information from the `mediasParams` in each model into the config file.

### CODE REVIEW

File: app/Models/CollectionObject.php [hunk 1]
```diff
@@ -51,30 +40,6 @@ class CollectionObject extends AbstractModel
...
-    /**
-     * A custom attribute that returns JSON, used for the Map field on the form.
-     * It is derived from the `latlon` attribute from the API resource.
-     */
-    public function getLatlngAttribute(): string
-    {
-        return json_encode([
-            'latlng' => Str::replace(',', '|', $this->latlon),
-        ]);
-    }
```
Comment (zachgarwood) [inline app/Models/CollectionObject.php:76]: The removal of the above two functions is just some unrelated minor cleanup.

**Review by trevinalonzo — APPROVED** [review]
perf!
